### DISCUSSION TITLE
Manifesto and Foundations

### DISCUSSION THREAD

**Discussion by anilturkmayali** [International-Data-Spaces-Association/Personal-Data-and-IDS#15] [Ideas]
Here please add your ideas on the manifesto and foundations of this group.

**Comment by mabuse68** [comment]
Which definition of "Personal Data" ?
For instance: 
[https://gdpr-info.eu/issues/personal-data/](https://gdpr-info.eu/issues/personal-data/)
[https://commission.europa.eu/law/law-topic/data-protection/reform/what-personal-data_en](https://commission.europa.eu/law/law-topic/data-protection/reform/what-personal-data_en)

It think that, with such a broad definition of personal data, we should mention that the workgroup isn't going to solve the problem of determining which data is personal, and which isn't. Although a data space would benefit from having such a mechanism, it isn't simply a practical idea.
I also think that, once we agree on which definition, we should mention it in the manifesto.

**Comment by mabuse68** [comment]
User- or Human-centric?
The manifesto promotes the "user-centric" approach.
However, I think the MyData document uses the better wording: "human-centric"; for an individual is not necessarily a user of a system.
